content/en/docs/concepts/architecture/nodes.md

```diff
@@ -424,20 +424,101 @@ for gracefully terminating normal pods, and the last 10 seconds would be
 reserved for terminating [critical pods](/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical).
 
 {{< note >}}
-When pods were evicted during the graceful node shutdown, they are marked as failed.
+When pods were evicted during the graceful node shutdown, they are marked as shutdown.
 Running `kubectl get pods` shows the status of the the evicted pods as `Shutdown`.
 And `kubectl describe pod` indicates that the pod was evicted because of node shutdown:
 
 ```
-Status:         Failed
 Reason:         Shutdown
 Message:        Node is shutting, evicting pods
 ```
 
-Failed pod objects will be preserved until explicitly deleted or [cleaned up by the GC](/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection).
-This is a change of behavior compared to abrupt node termination.
 {{< /note >}}
 
+### Pod Priority based graceful node shutdown {#pod-priority-graceful-node-shutdown}
+
+{{< feature-state state="alpha" for_k8s_version="v1.23" >}}
+
+To provide more flexibility during graceful node shutdown around the ordering
+of pods during shutdown, pod priority based graceful node shutdown feature
+allows cluster administers to explicitly define the ordering of pods during
+graceful node shutdown based on [priority classes](docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass).
+
+The [Graceful Node Shutdown](#graceful-node-shutdown) feature, as described
+above, shuts down pods in two phases, non-critical pods, followed by critical
+pods. If additional flexibility is needed to explicitly define the ordering of
+pods during shutdown in a more granular way, pod priority based graceful
+shutdown can be used.
+
+Pod Priority based graceful node shutdown makes it possible to do graceful node
+shutdown in multiple phases, each phase shutting down a particular priority
+class of pods. The kubelet can be configured with the exact phases and shutdown
+time per phase.
+
+Assuming the following custom pod [priority
+classes](docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass)
+in a cluster,
+
+|Pod priority class name|Pod priority class value|
+|-----------------------|------------------------|
+|custom-class-a         | 100000                 |
+|custom-class-b         | 10000                  |
+|custom-class-c         | 1000                   |
+|regular/unset          | 0                      |
+
+The [kubelet configuration](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration) `ShutdownGracePeriodByPodPriority` could look like:
+
+|Pod priority class value|Shutdown period|
```

Comment:
The column is correct, but I'm a little worried it might make it more confusing and might be better to explain the "greater than equal to next phase" logic in the actual text.